src/content/docs/guides/run-a-taiko-node.mdx

```diff
@@ -14,131 +15,181 @@ This guide will help you start up a Taiko RPC node using simple-taiko-node.
 - If using Windows, you should install [Git BASH](https://gitforwindows.org/) or [WSL](https://learn.microsoft.com/en-us/windows/wsl/install) to use as your terminal.
 - Meet the [Geth minimum hardware requirements](https://github.com/ethereum/go-ethereum#hardware-requirements) except for the storage requirement because Taiko nodes will require less storage (at the time of writing).
 
-## Run a Taiko node with simple-taiko-node
+## Run a Taiko node with `stn`
 
-### 1. Clone simple-taiko-node
 
-<Tabs>
-  <TabItem label="Mac/Linux">
-    ```sh
-    git clone https://github.com/taikoxyz/simple-taiko-node.git
-    cd simple-taiko-node
-    ```
-  </TabItem>
-  <TabItem label="Windows">
-    ```sh
-    git clone https://github.com/taikoxyz/simple-taiko-node.git
-    cd simple-taiko-node && git config core.autocrlf false
-    ```
-  </TabItem>
-</Tabs>
+<Steps>
 
-### 2. Copy the sample .env files
+1. Install `stn`:
```

Comment:
@brachsterX we've temporarily removed this part of the guide as stn hasn't yet been updated to support hekla

Reply:
yeah ill make sure to add it back once i add hekla support for stn. for now lets just use simple-taiko-node

Reply:
I've updated the code 👍

Reply:
@brachsterX also in enable-a-proposer guide :)

Reply:
I can't seem to change on my end or I may be doing something wrong @RogerLamTd D:

Reply:
> I can't seem to change on my end or I may be doing something wrong @RogerLamTd D:

Are you unable to just delete it?

Reply:
enable-a-proposer has may have been resolved but the conflict now is the  src/content/docs/guides/run-a-holesky-node.md file which is already deleted

Reply:
> enable-a-proposer may have been resolved but the conflict now is the src/content/docs/guides/run-a-holesky-node.md file which is already deleted

I see, could you merge main into your branch and resolve locally?

Reply:
Thanks for guiding me on this @RogerLamTd , however it is still giving the same conflict.

Reply:
@brachsterX it might be better to start fresh and cherry pick your changes to a new branch

Reply:
created a new one on #113 . I'll close this PR.